pkgs/top-level/all-packages.nix

```diff
@@ -39362,6 +39362,10 @@ with pkgs;
 
   fasthenry = callPackage ../applications/science/physics/fasthenry { };
 
+  fastcap = callPackage ../applications/science/physics/fastcap {
+    texlive = texlive.combine { inherit (texlive) scheme-medium; };
+  };
+
```

Comment:
New packages are going to [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name)

Reply:
Didn't realize that the change was implemented. I'll move the code.